docs/user-guide/configure.md

```diff
@@ -413,6 +413,18 @@ For example:
 
 [More info](options.md#ignoredisables).
 
+### `configurationComment`
```

Comment:
[ask] This new section is inserted under the `ignoreDisables` section, but is there any reason to do that?

Reply:
Felt related to disabling stuff but I wasn't sure the best place to put it so definitely open to moving it somewhere else that it fits better

Reply:
Let's move it to above `ignoreDisables` and make it a `##` heading.

Reply:
Updated